powerbi-docs/developer/visuals/bookmarks-support.md

```diff
@@ -46,20 +46,23 @@ The new selection state (or filter) is communicated through the `options.jsonFil
 
 ## Visuals with selection
 
+> [!NOTE]
+> InteractivityService has been deprecated.
+
 If your visual interacts with other visuals using [Selection](https://github.com/PowerBi-Projects/PowerBI-visuals/blob/master/Tutorial/Selection.md), you can add bookmark support in one of two ways:
 
-- Through [InteractivityService](#use-interactivityservice-to-restore-bookmark-selections) to manage selections, use the `applySelectionFromFilter`. This is the easier and preferred method.
-- Through [SelectionManager](#use-selectionmanager-to-restore-bookmark-selections), if your visual doesn't use **InteractivityService**.
+- Through [InteractivityService](#use-interactivityservice-to-restore-bookmark-selections) to manage selections, use the `applySelectionFromFilter`. This is deprecated method.
```

Comment:
Link needs to be changed to (#use-interactivityservice-to-restore-bookmark-selections---deprecated)

Reply:
Seems like PR has already been merged, do I need to create the new one with this fix?

Reply:
@kullJul Yes. I accidentally merged it. If you can create a new PR with this fix that would be great. Thanks.

Reply:
@mberdugo done -> https://github.com/MicrosoftDocs/powerbi-docs/pull/5124